packages/firebase_dynamic_links/test/firebase_dynamic_links_test.dart

```diff
@@ -497,5 +560,114 @@ void main() {
         ]);
       });
     });
+
+    group('onLink', () {
```

Comment:
test: I've added additional tests for checking `FirebaseDynamicLinks.instance.onLink(onSuccess: onSuccess, onError: onError)`